icu4c/source/common/unicode/docmain.h

```diff
@@ -158,6 +163,11 @@
  *     <td>icu::number::NumberRangeFormatter</td>
  *   </tr>
  *   <tr>
+ *     <td>Measure Unit Formatting</td>
+ *     <td>measunit.h</td>
+ *     <td>icu::MeasureUnit</td>
```

Comment:
These are just the units, not the formatting API. We already have "Number Formatting (includes currency and unit formatting)"

So I don't think we need this.

Reply:
Removed MeasureUnit